mobile/src/screens/chat-layout/chat.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import { ScrollView, Text, View } from '#libs/components/components';
+import {
+  useAppForm,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from '#libs/hooks/hooks';
+
+import { ChatInput, MessageItem } from './components/components';
+import { DEFAULT_VALUES, MOCKED_DATA, PREVIOUS_USER } from './libs/constants';
+import { styles } from './styles';
+
+type Message = {
+  id: number;
+  isUser: boolean;
+  message: string;
+};
+
+const Chat: React.FC = () => {
+  const { control, handleSubmit, reset } = useAppForm<{ text: string }>({
+    defaultValues: DEFAULT_VALUES,
+  });
+
+  const [messages, setMessages] = useState<Message[]>(MOCKED_DATA);
+  const scrollViewReference = useRef<ScrollView | null>(null);
+
+  const scrollViewToEnd = (): void => {
+    scrollViewReference.current?.scrollToEnd();
+  };
+
+  const onSubmit = useCallback(
```

Comment:
by quality criteria we should name methods inside component via handle
```suggestion
  const handleFormSubmit = useCallback(
```